apps/sms/js/thread_list_ui.js

```diff
@@ -402,10 +403,18 @@ var ThreadListUI = {
         thread.unreadCount = isRead ? 0 : 1;
         this.mark(thread.id, isRead ? 'read' : 'unread');
 
-        MessageManager.markThreadRead(thread.id, isRead);
+		threadToMark.push(thread.id);
       }
     });
 
+    this.markReadUnread(threadToMark, isRead);
+  },
+
+  markReadUnread: function thlui_markReadUnread(threadToMark, isRead) {
+    threadToMark.forEach((id) => {
+        MessageManager.markThreadRead(id, isRead);
```

Comment:
is identation incorrect here? though i am taking care of it, as line is getting >80 most of time, so i have to break it into lines, while taking care of identation at same time.